src/SamplesApp/SamplesApp.UITests/TestFramework/AppInitializer.cs

```diff
@@ -94,6 +94,7 @@ private static IApp CreateBrowserApp(bool alreadyRunningApp)
 					.ScreenShotsPath(TestContext.CurrentContext.TestDirectory)
 #if DEBUG
 					.Headless(false)
+					.SeleniumArgument("--remote-debugging-port=9222")
```

Comment:
Is it safe to put that here?  I mean it suppose we're not already having a browser in _debugging mode_...

Reply:
That's why I put it in DEBUG only. If you have any issue locally, you can comment it out, but usually I think it will be way more useful to have it enabled by default.